source/configuration_manual/sieve/pigeonhole_sieve_interpreter.rst

```diff
@@ -90,9 +90,9 @@ Extension
 `vnd.dovecot.environment <https://raw.githubusercontent.com/dovecot/pigeonhole/main/doc/rfc/spec-bosch-sieve-dovecot-environment.txt>`_         supported (v0.4.14+)                                                                                                        no                    Extends the standard "environment" extension with extra items and a variables namespace for direct access
 `vnd.dovecot.execute <https://raw.githubusercontent.com/dovecot/pigeonhole/main/doc/rfc/spec-bosch-sieve-extprograms.txt>`_                     :ref:`Extprograms Plugin <pigeonhole_plugin_extprograms>` (v0.3+)                                                           no                    Implements executing a pre-defined set of external programs with the option to process string data through the external program
 `vnd.dovecot.filter <https://raw.githubusercontent.com/dovecot/pigeonhole/main/doc/rfc/spec-bosch-sieve-extprograms.txt>`_                      :ref:`Extprograms plugin <pigeonhole_plugin_extprograms>` (v0.3+)                                                           no                    Implements filtering messages through a pre-defined set of external programs
-`vnd.dovecot.pipe <https://raw.githubusercontent.com/dovecot/pigeonhole/main/doc/rfc/spec-bosch-sieve-extprograms.txt>`_                        `Pipe Plugin <https://wiki.dovecot.org/Pigeonhole/Sieve/Plugins/Pipe>`_ (v0.2),                                             no                    Implements piping messages to a pre-defined set of external programs
+`vnd.dovecot.pipe <https://raw.githubusercontent.com/dovecot/pigeonhole/main/doc/rfc/spec-bosch-sieve-extprograms.txt>`_                        See :ref:`Extprograms plugin <pigeonhole_plugin_extprograms>`                                                               no                    Implements piping messages to a pre-defined set of external programs
                                                                                                                                                 :ref:`pigeonhole_plugin_extprograms` (v0.3+)
```

Comment:
Again a bit unrelated, but the table formatting makes it so, that the link to the `pigeonhole_plugin_extprograms` is duplicated. You probably need to remove the line that is mostly empty except for the "pigeonhole_plugin_extprograms" link.

Reply:
Fixed.